NoExecutablesPathChecker.cs

```diff
@@ -6,18 +6,22 @@ namespace PathCleaner
 {
     internal class NoExecutablesPathChecker : IPathChecker
     {
-        private static readonly HashSet<string> executableExtensions = new HashSet<string>()
+        private static readonly HashSet<string> executableExtensions = new HashSet<string>(System.StringComparer.OrdinalIgnoreCase)
         {
             ".exe", ".com", ".bat", ".cmd", ".ps1", ".dll"
         };
 
+        public NoExecutablesPathChecker()
+        {
+            // TODO: Make the class compatible with executable file extensions defined in the PATHEXT to prevent EmptyPathChecker from producing false positives.
+        }
+
         public string Reason => "No executables";
 
         public bool Identify(string folder, string previousFolder)
         {
             return !Directory.EnumerateFiles(folder)
-                .Where(s => executableExtensions.Contains(Path.GetExtension(s)))
-                .Any();
+                .Any(s => executableExtensions.Contains(Path.GetExtension(s)));
```

Comment:
This change is completely unnecessary. No improvement to readability or code speed.

Reply:
I thought it would improve readability but the decision is yours.